addons/project/security/project_security.xml

```diff
@@ -15,7 +15,7 @@
     <record id="group_project_manager" model="res.groups">
         <field name="name">Administrator</field>
         <field name="category_id" ref="base.module_category_services_project"/>
-        <field name="implied_ids" eval="[(4, ref('group_project_user')), (4, ref('mail.group_mail_template_editor'))]"/>
+        <field name="implied_ids" eval="[(4, ref('group_project_user')), (4, ref('mail.group_mail_template_editor')), (4, ref('mail.canned_response_administrator'))]"/>
```

Comment:
We should have these for each of these `canned_response_administrator` permissions in base code + with overrides.